android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -192,6 +214,37 @@ public void onNotified(RPCNotification notification) {
         }
     };
 
+    private final OnSystemCapabilityListener systemCapabilityListener = new OnSystemCapabilityListener() {
+        @Override
+        public void onCapabilityRetrieved(Object capability) {
+            VideoStreamingParameters params = new VideoStreamingParameters();
```

Comment:
@kostyaBoss I noticed a bug while testing RTP where on resize the protocol would change back to RAW (the default value)
```suggestion
           VideoStreamingParameters params = (parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
```

This fixes the issue as the previously set parameters will be used and then updated based on the capability received